server/src/controllers/game.controller.js

```diff
@@ -6,27 +6,35 @@ const response = require("../utils/response");
 const CustomError = require("../utils/custom-error");
 
 class GameController {
-  async create(req, res) {
-    try {
-      let game_id = uuid.v4();
-      let game_options = {
-        game_owner_user_id: uuid.v4(),
-        player_color: "b",
-      };
+    async create(req, res) {
+        try {
+            let game_id = uuid.v4();
+            var date_ob = new Date();
+            var game_owner_id = req.params['User_id']
+            var opponent_id = "playerY"
+            console.log(date_ob)
+            var new_game =  {
+                game_id: game_id,
+                game_owner_user_id: game_owner_id,
+                opponent_user_id: opponent_id,
+                start_time: date_ob,
+                end_time: null,
+                moves: [
+                    {game_owner_id:"//default_board_state"},
+                    {opponent_id:"//default_board_state"}
+                ],
+                result_id: null
+            }
 
-      // Save the new game in DB
-      // db code here
+            // Save the new game in DB
+            // db code here
```

Comment:
THis endpoint has already been implemented, your task was to persist the game created to the zc_core database